modules/thunder_article/config/install/core.entity_form_display.node.article.default.yml

```diff
@@ -91,8 +91,9 @@ content:
       title: Paragraph
       title_plural: Paragraphs
       edit_mode: preview
-      add_mode: dropdown
+      add_mode: modal
       form_display_mode: default
+      default_paragraph_type: _none
```

Comment:
If it's bug, then it will be **VERY** interesting:
https://www.drupal.org/node/2872958